datanode/gateway/graphql/schema.graphql

```diff
@@ -320,6 +322,8 @@ type ObservableMarketData {
   marketGrowth: String!
   "The last traded price (an unsigned integer)"
   lastTradedPrice: String!
+  "The current funding rate. This applies only to a perpetual market"
+  fundingRate: String
```

Comment:
**chore (non-blocking)** In the future, let's tidy this object up so that fields that don't apply to all markets don't apply to all markets. I've marked this as non blocking as there'll almost certainly be more than just this field. I'll file a ticket.